pipelines/noop.yaml

```diff
@@ -29,8 +29,8 @@ spec:
   tasks:
     - name: start 
       taskRef:
-        kind: ClusterTask
-        name: openshift-client
+        bundle: ${CONTAINER_REPO}/appstudio-tasks:utils-task-${TAG}
```

Comment:
there should not be this `bundle`, it will be automatically added when the name matches with the task name.